util/topgen.py

```diff
@@ -1184,10 +1184,9 @@ def render_template(template_path: str, rendered_path: Path, **other_info):
                            check=True,
                            cwd=str(SRCTREE_TOP))  # yapf: disable
 
-        # generate chip level xbar and alert_handler TB
+        # generate chip level xbar TB
         tb_files = [
-            "xbar_env_pkg__params.sv", "tb__xbar_connect.sv",
-            "tb__alert_handler_connect.sv", "xbar_tgl_excl.cfg"
```

Comment:
`xbar_tgl_excl.cfg`?

Reply:
Good call, thanks Sri, I did not push the right version..